webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultSchoolService.java

```diff
@@ -24,7 +24,7 @@ public DefaultSchoolService(@NotNull final SchoolRepository repository) {
 
     @Override
     public Set<Organization> getSchools(@NotNull final User user) {
-        final Permission permission = user.getPermissionsById().get("INDIVIDUAL_PII_READ");
+        final Permission permission = user.getPermissionsById().get("GROUP_READ");
```

Comment:
correct.  GROUP_READ is the correct permission.  I think we were leaving this till we decided it we wanted a new component.  We decided for now rdw_admin will use the 'reporting' component and it's either group_read to view groups or group_write to crud groups.